examples/channel-hello-world/ChannelHelloWorld_Client.cpp

```diff
@@ -27,20 +27,24 @@ int main(int argc, char* argv[]) {
   folly::SocketAddress address;
   address.setFromHostPort(FLAGS_host, FLAGS_port);
 
-  // create a client which can then make connections below
-  auto rsf = RSocket::createClient(
-      std::make_unique<TcpConnectionFactory>(std::move(address)));
-
-  auto rs = rsf->connect().get();
-
-  // send stream of strings to the server
-  rs->requestChannel(Flowables::justN({"initialPayload", "Bob", "Jane"})
-                         ->map([](std::string v) {
-                           std::cout << "Sending: " << v << std::endl;
-                           return Payload(v);
-                         }))
-      ->subscribe([](Payload p) {
-        std::cout << "Received: " << p.moveDataToString() << std::endl;
+  std::shared_ptr<RSocketClient> client;
+
+  RSocket::createConnectedClient(
```

Comment:
you can change this and all below to the same pattern with `get()` you used above